cartography/intel/gcp/bigtable.py

```diff
@@ -277,7 +279,7 @@ def _load_bigtable_clusters_tx(
         c.firstseen = timestamp()
     SET
         c.name = cluster.name,
-        c.location = cluster.name,
+        c.location = cluster.location,
```

Comment:
location seems to be in the format: `projects/{project}/locations/{zone}`. Let's create a new field called region where we store only the region part from zone value.

Reference: https://cloud.google.com/bigtable/docs/reference/admin/rest/v2/projects.instances.clusters#Cluster

Reply:
We need to store region from the location field. Read the zone and then find out the region from it. Can you check this again?

Reply:
In `get_bigtable_clusters` function i have build logic for this